log/sequencer.go

```diff
@@ -22,6 +23,11 @@ type Sequencer struct {
 	timeSource util.TimeSource
 	logStorage storage.LogStorage
 	keyManager crypto.KeyManager
+
+	// These parameters can be safely adjusted during operation
+	// sequencerGuardWindow when set to a non zero duration this establishes a cutoff point for new
+	// entries. Entries newer than the guard window will not be sequenced until they fall outside it.
```

Comment:
The last sentence is more helpful, so maybe just: 
```
// Entries newer than (now - sequencerGuardWindow) will not be sequenced yet.
```
and then add some explanation of why the guard window is needed.